docs/user_guide/config.md

```diff
@@ -194,152 +194,269 @@ This file adds:
+              - "443:443"
+            volumes:
+              - ./Caddyfile:/etc/caddy/Caddyfile
+              - caddy_data:/data
+              - caddy_config:/config
+
         secrets:
-          - db_admin_password
-          - db_user_password
-
-      federation:
-        image: "neurobagel/federation_api:${NB_FAPI_TAG:-latest}"
-        profiles:
-          - "local_federation"
-          - "full_stack"
-        ports:
-          - "${NB_FAPI_PORT_HOST:-8080}:8000"
-        volumes:
-          - "./local_nb_nodes.json:/usr/src/local_nb_nodes.json:ro"
-        environment:
-          NB_API_PORT: 8000
-          NB_FEDERATE_REMOTE_PUBLIC_NODES: ${NB_FEDERATE_REMOTE_PUBLIC_NODES:-True}
-          NB_ENABLE_AUTH: ${NB_ENABLE_AUTH:-false}
-          NB_QUERY_CLIENT_ID: ${NB_QUERY_CLIENT_ID}
-          VIRTUAL_HOST: myservice2.myinstitute.org # (3)!
-          LETSENCRYPT_HOST: myservice2.myinstitute.org # (4)!
-          VIRTUAL_PORT: 8000
-
-      query_federation:
-        image: "neurobagel/query_tool:${NB_QUERY_TAG:-latest}"
-        profiles:
-          - "local_federation"
-          - "full_stack"
-        ports:
-          - "${NB_QUERY_PORT_HOST:-3000}:5173"
-        environment:
-          NB_API_QUERY_URL: ${NB_API_QUERY_URL}
-          NB_QUERY_APP_BASE_PATH: ${NB_QUERY_APP_BASE_PATH:-/}
-          NB_ENABLE_AUTH: ${NB_ENABLE_AUTH:-false}
-          NB_QUERY_CLIENT_ID: ${NB_QUERY_CLIENT_ID}
-          VIRTUAL_HOST: myservice3.myinstitute.org # (5)!
-          LETSENCRYPT_HOST: myservice3.myinstitute.org # (6)!
-          VIRTUAL_PORT: 5173
-
-      nginx-proxy:
-        image: nginxproxy/nginx-proxy
-        container_name: nginx-proxy
-        ports:
-          - "80:80"
-          - "443:443"
-        volumes:
-          - ./config/nginx/reverse_proxy.conf:/etc/nginx/conf.d/reverse_proxy.conf
-          - vhost:/etc/nginx/vhost.d
-          - html:/usr/share/nginx/html
-          - certs:/etc/nginx/certs:ro
-          - /var/run/docker.sock:/tmp/docker.sock:ro
-
-      acme-companion:
-        image: nginxproxy/acme-companion
-        container_name: nginx-proxy-acme
-        volumes_from:
-          - nginx-proxy
+          db_admin_password:
+            file: ${NB_GRAPH_SECRETS_PATH:-./secrets}/NB_GRAPH_ADMIN_PASSWORD.txt
+          db_user_password:
+            file: ${NB_GRAPH_SECRETS_PATH:-./secrets}/NB_GRAPH_PASSWORD.txt
+
         volumes:
-          - certs:/etc/nginx/certs:rw
-          - acme:/etc/acme.sh
-          - /var/run/docker.sock:/var/run/docker.sock:ro
-
-    secrets:
-      db_admin_password:
-        file: ${NB_GRAPH_SECRETS_PATH:-./secrets}/NB_GRAPH_ADMIN_PASSWORD.txt
-      db_user_password:
-        file: ${NB_GRAPH_SECRETS_PATH:-./secrets}/NB_GRAPH_PASSWORD.txt
-
-    volumes:
-      graphdb_home:
-      vhost:
-      html:
-      certs:
-      acme:
-    ```
+          graphdb_home:
+          caddy_data:
+          caddy_config:
+        ```
+
+        ### Reverse Proxy Configuration with Caddy
```

Comment:
```suggestion
        **Reverse Proxy Configuration with Caddy**
```
If you turn this into a heading, the heading shows up on the right side in the Table of Contents, but it can only be clicked when after you select "CADDY" in the content tab above. That's a bit confusing, so instead let's just bold it or underline it or something like that